src/lib/components/common/Image.svelte

```
@@ -5,23 +5,27 @@
 
 <button
-	class={className}
+	class={`text-center ${className}`}
 	on:click={() => {
-		showImagePreview = true;
+		showImagePreview = false;
 	}}
 	type="button"
 >
 	<img src={_src} {alt} class={imageClassName} draggable="false" data-cy="image" />
+	{@html _caption}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)